contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -101,6 +101,7 @@ script: |
   export PATH=${WRAP_DIR}:${PATH}
 
   # Create the release tarball using (arbitrarily) the first host
+  export GIT_DIR="$PWD/.git"
```

Comment:
Since we build from the exported tarball, this is needed if we want the build system to use git to create the versioninfo. Otherwise, it will detect it is built from the tarball and not from git.